FreeIT/src/lesson3/Task17book.java

```diff
@@ -0,0 +1,34 @@
+package lesson3;
+
+public class Task17book {
+
+//	Создать двухмерный квадратный массив, и заполнить его «бабочкой»,
+
```

Comment:
You code works as expected.
My solution will be something like this:
for (int i = 0; i < array.length; i++) {
for (int j = 0; j < array.length; j++) {
if (i < array.length / 2) {
if (i <= j && array.length - i > j) {
array[i][j] = 1;
}
} else {
if (i >= j && array.length - i < j + 2) {
array[i][j] = 1;
}
}
}
}
please try to analyze. it uses twice less for circles.

Reply:
OK, thank you